docker/ftest-standalone-ping/pom.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>arquillian-cube-docker-parent</artifactId>
+        <groupId>org.arquillian.cube</groupId>
+        <version>1.0.0.Final-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>Arquillian Cube Docker Functional Test Standalone With PingPong</name>
+    <artifactId>arquillian-cube-docker-ftest-standalone-ping</artifactId>
```

Comment:
Rename module to a name that describes what it is testing; e.g. standalone-autostart